src/core/include/openvino/core/model.hpp

```diff
@@ -240,7 +248,12 @@ class OPENVINO_API Model : public std::enable_shared_from_this<Model> {
     /// when evaluating the model. This additional information can be shared across nodes.
     bool evaluate(ov::TensorVector& output_tensors,
                   const ov::TensorVector& input_tensors,
-                  ov::EvaluationContext evaluation_context = ov::EvaluationContext()) const;
+                  ov::EvaluationContext& evaluation_context) const;
+
+    /// \brief Evaluate the model on inputs, putting results in outputs.
+    /// \param output_tensors Tensors for the outputs to compute. One for each result
+    /// \param input_tensors Tensors for the inputs. One for each inputs.
+    bool evaluate(ov::TensorVector& output_tensors, const ov::TensorVector& input_tensors) const;
```

Comment:
how these changes are related to ov::Any?

Reply:
Hi
Previously Model::evaluate took evaluation_context by value. It works only if ov:Any has broken copy constructor.
Model::evaluate is broken due to ov::Any copy fix.

So I changed Model::evaluate declaration to take evaluation_context  by reference.